index.js

```diff
@@ -13,7 +13,9 @@ var defaultOptions = {
         '`': '&#x60;',
         '=': '&#x3D;',
     },
+    encodeDelimiters: ['{{{', '}}}'],
```

Comment:
This is duplicated information. It seems `encodeDelimiters` this belongs inside of `defaultTags.encoded`

Reply:
Indeed so.